pkg/volume/azure_dd/azure_common.go

```diff
@@ -175,157 +174,6 @@ func (handler *osIOHandler) Readlink(name string) (string, error) {
 	return os.Readlink(name)
 }
 
-func (handler *osIOHandler) ReadFile(filename string) ([]byte, error) {
-	return ioutil.ReadFile(filename)
-}
```

Comment:
got removed?

Reply:
It's a code merge issue, get it back now